src/core/derivation.ts

```diff
@@ -260,11 +294,13 @@ export function clearObserving(derivation: IDerivation) {
 }
 
 export function untracked<T>(action: () => T): T {
+    const prevAllowStateReads = allowStateReadsStart(false)
```

Comment:
Same as `transaction`: https://github.com/mobxjs/mobx/pull/2079/files#r327233937

Reply:
Note that i set it here to false,
I've added that due your comment on the pre iteration,
its more correct that you will get warnings inside `untracked`

Reply:
I missed the `false` value, nevertheless I still think it shouldn't switch the flag.
Untracked is used to disable subscription in any context ... if we disable reads inside untracked it becomes absolutely useless (because you're either outside or reaction so it has nothing to disable or it will report error).
The idea is that if you use `untracked` from `action`/`reaction`, `globalState.allowStateReads` is already `true`, so there is no error/exception. Which seems correct to me, because the possible error (missing subscription) is introduced explicitely and intentionally. 
Previously this wasn't possible, because reporting was bound to `globalState.trackingDerivation`. That was analogical to `enforceAction` checking for `inBatch` flag, rather than `allowStateChanges`... therefore my comment and introduction of `allowStateReads`...

Reply:
done